ElectronClient/app/gui/TagList.jsx

```diff
@@ -0,0 +1,48 @@
+const React = require('react');
```

Comment:
Added TagList and TagItem components as we can _do more_ with the tags under the note in the future.